server.js

```diff
@@ -99,14 +103,56 @@ function handleWeather(request, response) {
 //   }
 }
 
-
-
-
 function Weather(data) {
   this.time = data.time;
   this.forecast = data.summary;
 }
 
+
+
+
+// SQL locations
+
+app.get('/location', (req,res) => {
+  
+  const SQL = 'SELECT * FROM location';
+  
+  client.query(SQL)
+    .then( results => {
+      if( results.rowCount >= 1 ) {
+        res.status(200).json(location);
+      }
```

Comment:
With your current logic, this would always send the same information regardless of what you searched for due to how you wrote your SQL query above.